app/services/stage_globus.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Moves files from the ActiveStorage to the staging mount
+class StageGlobus
```

Comment:
Worth adding unit tests for `StageGlobus`? Maybe you can pull one or more out of the `StageBlobs` spec?